docs/specs/build/config/basics.yml

```diff
@@ -147,7 +147,7 @@ inputs:
   docs/a.md:
   docs/b/b.md:
 outputs:
-  output/a.json:
+  output.json:
```

Comment:
tend to make the value only contains only output folder. URL is confusing when considering pretty URL or extension.